cli/bptest/metadata_lint.go

```diff
@@ -0,0 +1,126 @@
+package bptest
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+
+	"gopkg.in/yaml.v2"
+)
+
+// BlueprintConnection structure to match metadata YAML
+type BlueprintConnection struct {
```

Comment:
Can't we reuse the proto in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/tree/master/cli/bpmetadata/proto?

Reply:
Done